packages/itwin/grouping-mapping-widget/src/components/Mappings/hooks/useFetchMappingExtractionStatus.ts

```diff
@@ -35,13 +35,13 @@ export const fetchMappingStatus = async (
     switch(extractionStatusResponse.state) {
```

Comment:
Need to add handling for `PartiallySucceeded` as well.

Reply:
Handled.